examples/01-multi-time-series-and-covariates.ipynb

```diff
@@ -180,19 +180,18 @@
     "* The **target time series** is the time series we are interested to forecast (given its history)\n",
     "* A **covariate time series** is a time series which may help in the forecasting of the target series, but that we are not interested in forecasting. It's sometimes also called *external data*.\n",
     "\n",
+    "**Note**: Static covariates are invariant in time and correspond to additional information associated with the components of a `TimeSeries`. Detailed information about this kind of covariate can be found in the [static covariates notebook](https://unit8co.github.io/darts/examples/15-static-covariates.html).\n",
```

Comment:
associated with the components of the **target time series**.

Reply:
Can't they also be part of the covariates series?

Reply:
They can be embedded into the covariates but will not be used by any model.
Static covariates are characteristics of the target series.